tests/common/reboot.py

```diff
@@ -277,6 +281,12 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
         if not ret:
             raise Exception('warmboot-finalizer service timeout on DUT {}'.format(hostname))
 
+    # Verify if the temporary file created in tmpfs is deleted after reboot, to determine a
+    # successful reboot
+    file_check = duthost.command('find /dev/shm/test_reboot', module_ignore_errors=True)['rc']
+    if not file_check:
+        raise Exception('DUT {} did not reboot'.format(hostname))
```

Comment:
It is best to remove the file (cleanup) that you created at this point when the test has failed.

Reply:
Incorporated the change. Thank you.

Reply:
@vvolam checked with Vaibhav, don't delete this file explicitly for debugging purpose